README.md

```diff
@@ -224,16 +224,19 @@ yarn          # Install dependencies and sets up workspces
 yarn build    # Builds contracts, clients, etc
 
 # starting network
-yarn run:network
+HOPR_ENVIRONMENT_ID=hardhat-localhost yarn run:network
+
+# workaround for a temp issue with local hardhat-network
+cp -R packages/ethereum/deployments/hardhat-localhost/localhost packages/ethereum/deployments/hardhat-localhost/hardhat
```

Comment:
where can I read more on this issue?

Reply:
https://github.com/hoprnet/hoprnet/issues/2907